packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx

```diff
@@ -7,6 +7,7 @@ import { Arrow } from '@/common/components/icons'
 import { ListItem } from '@/common/components/List'
 import { RowGapBlock } from '@/common/components/page/PageContent'
 import { ProgressBar } from '@/common/components/Progress'
+import { Tooltip, TooltipDefault } from '@/common/components/Tooltip'
```

Comment:
`TooltipDefault` import is not used, can we remove this one?
<img width="774" alt="image" src="https://user-images.githubusercontent.com/9252017/219491155-ddf521fd-bd08-4fa7-93c2-2d3532405116.png">

```suggestion
import { Tooltip } from '@/common/components/Tooltip'
```